src/views/settings.html

```diff
@@ -3,168 +3,155 @@
-              <input
-                type="checkbox"
-                class="form-check-input"
-                name="configEncrypted"
-                id="settings_configEncrypted"
-              />
-              <label class="form-check-label" for="settings_configEncrypted">
-                Encrypted
-              </label>
-            </div>
-          </li>
-          <li class="list-group-item">
-            <div class="form-group mb-0">
-              <label class="mb-1" for="settings_configEncryptionKey"
-                >Encryption Key</label
-              >
-              <input
-                type="text"
-                class="form-control"
-                name="configEncryptionKey"
-                id="settings_configEncryptionKey"
-              />
-            </div>
-          </li>
-          <li class="list-group-item">
-            <div class="form-check">
-              <input
-                type="checkbox"
-                class="form-check-input"
-                name="autoUpdateConfig"
-                id="settings_autoUpdateConfig"
-              />
-              <label class="form-check-label" for="settings_autoUpdateConfig">
-                Refresh Automatically
-              </label>
-            </div>
-          </li>
-          <li
-            class="
-              list-group-item
-              d-flex
-              justify-content-between
-              align-items-center
-            "
-          >
-            <div>
-              <label class="mb-1">Last Refreshed on:</label>
-              <div id="settings_lastConfigUpdate"></div>
-              <div
-                class="text-danger"
-                id="settings_lastConfigUpdateError"
-              ></div>
-            </div>
+              {{!-- The input fields --}}
+              {{#isInput}}
+                <li class="list-group-item">
+                  <div class="form-group mb-0">
+                    <label class="mb-1" for="settings_{{key}}"
+                      >{{label}}</label
+                    >
+                    <input
+                      type="text"
+                      class="form-control"
+                      name="{{key}}"
+                      id="settings_{{key}}"
+                    />
+                  </div>
+                </li>
+              {{/isInput}}
+
+              {{!-- The text fields --}}
+              {{#isText}}
+              <li
+                  class="
+                    list-group-item
+                    d-flex
+                    justify-content-between
+                    align-items-center
+                  "
+                >
+                <div>
+                  <label class="mb-1">{{label}}</label>
+                  <div id="settings_{{key}}"></div>
+                  <div
+                    class="text-danger"
+                    id="settings_{{key}}Error"
+                  ></div>
+                </div>
+
+              {{/isText}}
```

Comment:
If `lastConfigUpdate` is the **only** item of type `text`, then maybe we shouldn't include it on the Items loops. We can just put it outside, like (note this is pseudo-code):

```
{{/items}}

<div>
  <label class="mb-1">Last Refreshed on:</label>
   <div>{{ lastConfigUpdate || "Never" }}</div>
     {{#lastConfigUpdateError}}
         <div class="text-error">{{ lastConfigUpdateError }}</div>
     {{/lastConfigUpdateError}}
</div>
<div>
  <button
    type="button"
    class="btn btn-success"
    id="settings_refreshNow"
  >
    <i class="fas fa-sync"></i> Refresh Now
  </button>
</div>
```